azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/_constants.py

```diff
@@ -15,7 +15,9 @@
 
 __LOG_PATH_LINUX = "/var/log/applicationinsights"
```

Comment:
Nit: With an underscore in front of `_constants.py`, no need for double underscores, just single is fine.